src/io/io.cpp

```diff
@@ -6,9 +6,11 @@
 
 #include <algorithm>
 #include <ctime>
+#include <filesystem>
 #include <fstream>
 #include <iostream>
 #include <string>
+#include <vector>
```

Comment:
Basic question, what is <vector> providing in this PR?

Reply:
that was an oversight. I thought I removed it. I'll fix it now